src/components/Onboarding.jsx

```diff
@@ -0,0 +1,144 @@
+import * as rb from 'react-bootstrap'
+import React, { useState } from 'react'
+import Sprite from './Sprite'
+import { useSettingsDispatch } from '../context/SettingsContext'
+
+export default function Onboarding() {
+  const [step, setStep] = useState(0)
+  const settingsDispatch = useSettingsDispatch()
+
+  if (step === 0) {
+    return (
+      <div class="mx-auto my-auto py-5">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="logo" width="200" height="200" className="mb-4" />
+        </div>
+        <h1 class="text-center">JoinMarket</h1>
+        <p class="text-center fs-4 text-secondary mb-5">Top-notch privacy for your bitcoin.</p>
+        <div class="d-flex justify-content-center mb-2">
+          <rb.Button size="lg" className="col-10" variant="dark" onClick={() => setStep(1)}>
+            Get started
+          </rb.Button>
+        </div>
+        <div class="d-flex justify-content-center">
+          <rb.Button variant="dark" onClick={() => settingsDispatch({ showOnboarding: false })}>
+            Skip intro
+          </rb.Button>
+        </div>
+        <p class="mt-4 text-center">
+          Your wallet, your coins. <br />
+          100% open-source & open-design.
+        </p>
+      </div>
+    )
+  } else if (step === 1) {
+    return (
+      <div class="mx-auto my-auto py-5 col-10 col-md-4">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="welcome" width="150" height="150" className="mb-4" />
+        </div>
+        <p class="fs-3 text-center">1: Welcome to JoinMarket!</p>
+        <p class="text-center text-secondary mb-5">
+          JoinMarket is a privacy-focused software solution that aims to improve the confidentiality and privacy of your
+          bitcoin transactions. It facilitates the creation of collaborative transactions through a peer-to-peer
+          marketplace. <br />
+          <br />
+          ⚠️ Warning: While JoinMarket is tried and tested, this user interface is not. If anything breaks please{' '}
+          <a
+            href="https://github.com/joinmarket-webui/joinmarket-webui/issues"
+            target="_blank"
+            rel="noreferrer"
+            class="link-secondary"
+          >
+            report an issue
+          </a>{' '}
+          on GitHub.
+        </p>
+        <div class="d-flex justify-content-center">
+          <rb.Button size="lg" variant="dark" onClick={() => setStep(2)}>
```

Comment:
Should we make them those a bit larger as well?

```suggestion
          <rb.Button size="lg" className="col-10" variant="dark" onClick={() => setStep(2)}>
```